lib/data-helpers/custom-parameter.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const { struct } = require('@janiscommerce/superstruct');
+const ApiListError = require('../api-list-error');
+
+const { mapperFilter } = require('../helpers/mapper-filter');
+
+const apiCustomParameterStruct = {
+	customParameters: ['string|object']
+};
+
+module.exports = class ListCustomParameters {
+
+	constructor(requestParameters, customParameters) {
+
+		this.validate({ customParameters });
+
+		this.requestParameters = requestParameters;
+		this.customParameters = customParameters.map(customParameter => {
+			return typeof customParameter === 'object' ? customParameter : { name: customParameter };
+		});
+	}
+
+	validate(customParameters) {
```

Comment:
Este metodo tampoco va a necesitar parametros porque puede tomarlo del `this.customParameters` 😌